app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,74 @@
+require_relative '../../lib/TunesTakeoutWrapper'
+
+class SuggestionsController < ApplicationController
+
+# index: shows top 20 suggestions, ranked by total number of favorites
+  def index
+    if current_user
+      favorites = TunesTakeoutWrapper.favorites(current_user.uid)
+      # array of pairs user has already favorited
```

Comment:
Style comment: this is a helpful comment that provides context, but would be better if it was above line 7 instead of below it